deploy/managed-velero-operator/100-velero.Namespace.yaml

```diff
@@ -1,6 +0,0 @@
-apiVersion: v1
-kind: Namespace
```

Comment:
I think deleting this will delete the NS and the Velero CRs

Reply:
I tested in hive-stage, and it appears to keep the namespace and CR around when the SelectorSyncSet is deleted.